src/ManifestStrategy/GlossaryManifestStrategy.php

```diff
@@ -149,11 +149,21 @@ protected function createGlossaryFileLine(string $glossaryKey, string $glossaryK
     {
         return implode(',', [
             $glossaryKey,
-            $glossaryKeyValue,
+            $this->wrapInQuotes($glossaryKeyValue),
             $glossaryKeyLanguage,
         ]);
     }
 
+    /**
+     * @param string $string
+     *
+     * @return string
+     */
+    protected function wrapInQuotes(string $string): string
+    {
+        return sprintf('"%s"', $string);
+    }
```

Comment:
Well, if we add the quotes always, it will decrease our manifest rating, because usually the new values are added without quotes. 

I think it makes more sense to check if the value contains comma, and in this case only add the quotes.

Reply:
I understand that we check lines. If the original change was without quotes, and the new one will be with quotes, we get 2 different lines

Reply:
It was fixed